README.md

```diff
@@ -23,26 +23,24 @@ This SDK requires native browser Javascript and has been tested in all modern br
 | ------ | ------- | ------ | ---------- |
 |    &#10004;   |    &#10004;    |   &#10004;    |  9, 10, 11 |
 
-### API Key
+### API Key (formerly App ID)
```

Comment:
Haha, I'll do a txt search thru the whole code base for anything api-key, api key, etc